specification/cognitiveservices/data-plane/Speech/VideoDubbing/preview/2024-05-20-preview/video-dubbing.json

```diff
@@ -0,0 +1,849 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a project": {
+            "$ref": "./examples/create_content_editing.json"
+          }
+        }
+      }
+    },
+    "/translations/{id}/edit-operations": {
+      "get": {
+        "tags": [
+          "EditOperations"
+        ],
+        "description": "Gets edit operations list info.",
+        "operationId": "EditOperations_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/PathResourceId"
+          },
+          {
+            "$ref": "#/parameters/QueryApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/PaginatedTranslations"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Operation": {
+            "$ref": "./examples/get_edit_operations.json"
+          }
+        }
+      }
+    },
+    "/translations/{id}/edit-operations/{operation-id}": {
+      "get": {
+        "tags": [
+          "EditOperation"
+        ],
+        "description": "Gets edit operation info.",
+        "operationId": "EditOperations_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/PathResourceId"
+          },
+          {
+            "$ref": "#/parameters/PathOperationId"
+          },
+          {
+            "$ref": "#/parameters/QueryApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/Translation"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Operation": {
+            "$ref": "./examples/get_edit_operation.json"
+          }
+        }
+      }
+    },
+    "/translate-operations/{operation-id}": {
```

Comment:
Why not merge with /delete-operations/{operation-id}?

Reply:
Because it will response translated result.